runtime/tests/integration_test.rs

```diff
@@ -212,7 +212,8 @@ fn transfer_to_stake() {
 				None
 			))
 			.dispatch(origin_of(AccountId::from(BOB))));
-			assert_eq!(Balances::free_balance(AccountId::from(BOB)), 2_000 * GLMR,);
```

Comment:
@JoshOrndorff this assert passing actually shows that the transfer through the EVM did not succeed because BOB has the same balance as before

Reply:
never mind, im dumb